torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -645,7 +645,32 @@ def rotate(
         return rotate_image_pil(inpt, angle, interpolation=interpolation, expand=expand, fill=fill, center=center)
 
 
-pad_image_pil = _FP.pad
+def _parse_pad_padding(padding: Union[int, List[int]]) -> List[int]:
+    if not isinstance(padding, (int, tuple, list)):
+        raise TypeError(f"`padding` should be an integer or tuple or list of integers, but got {padding}")
+
+    if isinstance(padding, int):
+        if torch.jit.is_scripting():
+            # This maybe unreachable
+            raise ValueError("padding can't be an int while torchscripting, set it as a list [value, ]")
```

Comment:
I'm probably missing something, we've never used this idiom anywhere else before. Why can't we just remove these lines and let the follow link handle it? Do we get JIT errors?

Reply:
TBH, I copy-pasted this. Let me give it a try.

Reply:
This error was actually harmful. We had some xfails in place that triggered on that rather then on a JIT failure. So we actually do support integer paddings while scripting.